101-data-factory-blob-to-sql-copy/azuredeploy.json

```diff
@@ -0,0 +1,222 @@
+  {
+    "contentVersion": "1.0.0.0",
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#", 
+    "parameters": {
+      "dataFactoryName": {
+        "type": "string",
+        "metadata": {
+          "description":  "Data factory name"
+        }
+      },
+      "storageAccountName": {
+        "type": "string",
+        "metadata": {
+          "description":  "Azure Storage account name"
+        }
+      },
+      "storageAccountKey": {
+        "type": "securestring",
+        "metadata": {
+          "description":  "Azure Storage account key"
+        }
+      },
+      "sourceBlobContainer": {
+        "type": "string",
+        "metadata": {
+          "description":  "Azure blob container name"
+        }
+      },
+      "sourceBlobName": {
+        "type": "string",
+        "metadata": {
+          "description":  "Azure blob name"
+        }
+      },
+      "sqlServerName": {
+        "type": "string",
+        "metadata": {
+          "description":  "Azure SQL server name"
+        }
+      },
+      "databaseName": {
+        "type": "string",
+        "metadata": {
+          "description":  "Azure SQL database name"
+        }
+      },
+      "sqlServerUserName": {
+        "type": "string",
+        "metadata": {
+          "description":  "Azure SQL server - user name"
+        }
+      },
+      "sqlServerPassword": {
+        "type": "securestring",
+        "metadata": {
+          "description":  "Azure SQL server - user password"
+        }
+      },
+      "targetSQLTable": {
+        "type": "string",
+        "metadata": {
+          "description":  "Azure SQL table name"
+        } 
+      }
+    },
+    "variables": {
+      "apiVersion": "2015-10-01",
```

Comment:
remove variables and use literal values for apiVersion so schema validation can be done.